### PR TITLE
feature/1275_update_doc_regarding_geolocation_types_in_carto

### PR DESCRIPTION
**Pull request by frbattid**
* Partially implements issue #1275 
    * No CHANGES_NEXT_RELEASE update is required, since the issue was re-opened

### CODE REVIEW

File: doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_cartodb_sink.md [hunk 1]
```diff
@@ -92,9 +92,10 @@ A single insert is composed for each notified entity, containing such insert the
...
 * `entityId`: Notified entity identifier.
 * `entityType`: Notified entity type.
-* `the_geom`: The current geolocation point, i.e. containing longitude and latitude. It must be of type PostGIS Geometry, it can be created for instance using [`ST_Point()`](http://postgis.net/docs/ST_Point.html). The data used for composing the PostGIS Geometry is obtained from a special notified attribute:
-    * Either of type `geo:point`.
-    * Either having associated a `location` metadata of type `string` and value `WGS84`.
+* `the_geom`: The current geolocation point, i.e. containing longitude and latitude. It must be of type PostGIS Geometry, but it must only contain points. It can be created for instance using [`ST_Point()`](http://postgis.net/docs/ST_Point.html). The data used for composing the PostGIS Geometry is obtained from a special notified attribute:
+    * Either of type `geo:point` (a point).
+    * Either of type `geo:json` (a GeoJson representing a point).
+    * Either having associated a `location` metadata of type `string` and value `WGS84`. Please observe this option is deprecated.
```
Comment (fgalan) [inline doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_cartodb_sink.md]: Maybe "this option is deprecated" could be a link to https://fiware-orion.readthedocs.io/en/master/deprecated/index.html

File: doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_cartodb_sink.md [hunk 2]
```diff
@@ -519,12 +521,14 @@ When required, the Annex 1 shows how to provision a table for Carto, among other
...
 ####<a name="section2.3.6"></a>Supported Orion's geometries
-Current version of `NGSICartoDBSink` supports the following Orion's geometries:
+Current version of `NGSICartoDBSink` supports the following NGSIv2 geometries:
 
 * `geo:point`, in this case the geolocated attribute is about a single point.
-* `geo:json`, in this case the geolocated attribute is about any geometry, from a simple point to a complex polygon.
+* `geo:json`, despite a GeoJson can describe any geometry, from a simple point to a complex polygon, it must represent a single point.
 * `location` metadata, in this case the geolocated attribute is about a single point.
 
+You can get more information at [NGSIv2]() specification (<i>Geospatial properties of entities</i> section).
```
Comment (fgalan) [inline doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_cartodb_sink.md]: Actual link to NGSIv2 spec missing.

File: doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_cartodb_sink.md [hunk 3]
```diff
@@ -112,9 +113,10 @@ A single insert is composed for each notified entity, containing such insert the
...
 * `entityId`: Notified entity identifier.
 * `entityType`: Notified entity type.
-* `the_geom`: The current geolocation point, i.e. containing longitude and latitude. It must be of type PostGIS Geometry, it can be created for instance using [`ST_Point()`](http://postgis.net/docs/ST_Point.html). The data used for composing the PostGIS Geometry is obtained from a special notified attribute:
-    * Either of type `geo:point`.
-    * Either having associated a `location` metadata of type `string` and value `WGS84`.
+* `the_geom`: The current geolocation point, i.e. containing longitude and latitude. It must be of type PostGIS Geometry, but it must only contain points. It can be created for instance using [`ST_Point()`](http://postgis.net/docs/ST_Point.html). The data used for composing the PostGIS Geometry is obtained from a special notified attribute:
+    * Either of type `geo:point` (a point).
+    * Either of type `geo:json` (a GeoJson representing a point).
+    * Either having associated a `location` metadata of type `string` and value `WGS84`. Please observe this option is deprecated.
```
Comment (frbattid) [inline doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_cartodb_sink.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1287/commits/ccdf187fadde5a966c09b2a43622c88471c8ecec

File: doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_cartodb_sink.md (hunk 2 above)
Comment (frbattid) [inline doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_cartodb_sink.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1287/commits/ccdf187fadde5a966c09b2a43622c88471c8ecec

**Comment by fgalan** [comment]
LGTM

**Pull request merged**